docs/workflows/data/file-write.md

```diff
@@ -0,0 +1,50 @@
+flowclass: Edge 1.0.0
+description: Learn more about the Losant Write: Read Node.
+
+# File Write
+
+The File Write Node allows you to write to a file on your [link](Edge Compute device). This node is *only* available in workflows for your edge device agent.
+
+![File Write Node](/images/workflows/data/file-write-node.png "File Write Node")
+
+## Configuration
+
+There are four configuration settings for this node - file path template, encoding, file content, how to handle writing the file if it exists. All fields expect for encoding are [templatable](/workflows/accessing-payload-data/#string-templates) and can be pulled off of the current workflow payload.
```

Comment:
would it make sense to quote the configuration settings? "file path template", "encoding", "file content"...